doc/WAW_dialog_doc.txt

```diff
@@ -8,6 +8,12 @@
 	Instead of <xs:all> tag this schema uses <xs:sequence minOccurs="1" maxOccurs="1"> tag which allows "unbounded"
 	value of "maxOccurs" attribute of inside tags.
 
+	`scope` attribute indicates whether to include this tag to output dialog or not.
+	- this node will be included only if official czech dialog is built:
```

Comment:
I wold say 'following' rather then 'this' as it is mentioned before.

Reply:
I slightly extended the documentation, further improvements will come with following issues (#66)